config/development.js

```diff
@@ -1,4 +1,7 @@
 const { deferConfig } = require('config/defer')
+const winston = require('winston')
+
+winston.level = 'debug'
 
 module.exports = {
   'pubsweet-server': {
```

Comment:
Don't you need to add the logger to `pubsweet-server.logger` as per https://gitlab.coko.foundation/pubsweet/pubsweet/tree/master/packages/logger#usage

Reply:
this is already done in `default.js`

Reply:
OK, I had my doubts that logger will be the same object, but I just tested this and it seems to be the case.